proofer.gemspec

```diff
@@ -27,7 +27,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.3.0'
```

Comment:
Forgot to mention you should prob upgrade the ruby dependency here as well

Reply:
good catch, I'll fix